CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ Please visit [cucumber/CONTRIBUTING.md](https://github.com/cucumber/cucumber/blo
   ([#1329](https://github.com/cucumber/cucumber-ruby/pull/1329)
    [deivid-rodriguez](https://github.com/deivid-rodriguez))
 
+* Make SIGINT/`Ctrl+c` behavior consistent with SIGTERM/`kill` behavior - now first invocation causes existing scenario to stop running and jump to `at_exit`, second invocation causes immediate exit. Before that first invocation only instructed Cucumber to exit after scenario and second invocation caused immediate exit skipping `at_exit`.
```

Comment:
Tiny formatting detail: if you remove the newline at line 28, the bullet list becomes 1 bullet list, making formatting nicer.

Reply:
Just git commit amend this @akostadinov and I'll approve and merge this in.

Reply:
Didn't notice that! fixed now